### ISSUE TITLE
Layout animation + MapView

### ISSUE THREAD

**Issue by rahmanmarija** [react-community/react-native-maps#1658]
Is it possible to use Layout Animation with MapView? For example to animate height of map, but using native animations.

**Comment by kytwb** [comment]
@rahmanmarija if you use `MapView.Animated`, then I guess yes.

**Comment by jerryheir** [comment]
@kytwb No one answers a question better or faster than a collaborator. I stumbled on this right now while looking for a way to disable the animation in MapView. Any help?

**Comment by kytwb** [comment]
@jerryheir The question is not related to `react-native-maps`. It's related to React Native's own [Animated](https://facebook.github.io/react-native/docs/animated) usage. I invited the person to open a thread on StackOverflow not to pollute the repository issues with questions unrelated to the module.

> disable the animation in MapView

This is different from what the original poster asked and I'm not sure what animation you are making reference to. Looking at the [MapView](https://github.com/react-native-community/react-native-maps/blob/master/docs/mapview.md) documentation and doing a quick CTRL+F on `animation`, I found `setCamera`: 

> Like animateCamera, but sets the new view instantly, without an animation.

Hope that helps.
